pkg/scheduler/framework/plugins/noderesources/least_allocated.go

```diff
@@ -21,11 +21,11 @@ import (
 )
 
 // leastResourceScorer favors nodes with fewer requested resources.
-// It calculates the percentage of memory and CPU requested by pods scheduled on the node, and
+// It calculates the percentage of memory, CPU and other reources requested by pods scheduled on the node, and
 // prioritizes based on the minimum of the average of the fraction of requested to capacity.
 //
 // Details:
-// (cpu((capacity-sum(requested))*MaxNodeScore/capacity) + memory((capacity-sum(requested))*MaxNodeScore/capacity))/weightSum
+// (cpu((capacity-sum(requested))*MaxNodeScore/capacity*cpuWeight) + memory((capacity-sum(requested))*MaxNodeScore/capacity)*memoryWeight + ...)/weightSum
```

Comment:
in your ways, `cpu((capacity-sum(requested))*MaxNodeScore/capacity*cpuWeight)` should be `cpu((capacity-requested)*MaxNodeScore*cpuWeight/capacity)`

WDYT?